source/index.js

```diff
@@ -401,12 +401,10 @@ class Storyblok {
           this.resolveAssetsRelations(response.data)
         }
 
-        if (res.headers["per-page"]) {
-          response = Object.assign({}, response, {
-            perPage: parseInt(res.headers["per-page"]),
-            total: parseInt(res.headers["total"]),
-          });
-        }
+        response = Object.assign({}, response, {
+          perPage: res.headers["per-page"] ? parseInt(res.headers["per-page"]) : null,
```

Comment:
@fernandacanto Could you replace this null return to a zero number? Returning a null can raise an unknown sort of problems.

Reply:
Same here, please.